### PR TITLE
[ShopBundle] Unification of service names

### PR DESCRIPTION
**Pull request by Wojdylak**
| Q               | A
|-----------------|-----
| Branch?         | 1.14
| Bug fix?        | no
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | yes
| Related tickets | #17093
| License         | MIT

### CODE REVIEW

File: UPGRADE-1.14.md [hunk 1]
```diff
@@ -115,6 +115,38 @@
...
+    | `sylius.controller.shop.locale_switch`                                                                     | `sylius_shop.controller.locale_switch`                                               |
+    | `sylius.controller.shop.security_widget`                                                                   | `sylius_shop.controller.security_widget`                                             |
+    | `sylius.controller.shop.register_thank_you`                                                                | `sylius_shop.controller.register_thank_you`                                          |
+    | `sylius.mailer.contact_email_manager.shop`                                                                 | `sylius_shop.mailer.contact_email_manager`                                           |
+    | `sylius.mailer.order_email_manager.shop`                                                                   | `sylius_shop.mailer.order_email_manager`                                             |
```
Comment (NoResponseMate) [inline UPGRADE-1.14.md]:
```suggestion
    | `sylius.mailer.contact_email_manager.shop`                                                                 | `sylius_shop.email_manager.contact`                                           |
    | `sylius.mailer.order_email_manager.shop`                                                                   | `sylius_shop.email_manager.order`                                             |
```

File: UPGRADE-1.14.md [hunk 2]
```diff
@@ -115,6 +115,38 @@
...
     | `sylius.promotion_coupon_generator.percentage_policy`                                                      | `sylius.generator.percentage_generation_policy`                                      |
+    | **ShopBundle**                                                                                             |                                                                                      |
+    | `sylius.shop.locale_switcher`                                                                              | `sylius_shop.locale_switcher`                                                        |
+    | `sylius.storage.locale`                                                                                    | `sylius_shop.locale_storage`                                                         |
+    | `sylius.context.locale.storage_based`                                                                      | `sylius_shop.locale_context.storage_based`                                           |
```
Comment (NoResponseMate) [inline UPGRADE-1.14.md]:
```suggestion
    | `sylius.context.locale.storage_based`                                                                      | `sylius_shop.context.locale.storage_based`                                           |
```
It's pretty much a global convention for contexts

File: src/Sylius/Bundle/ShopBundle/Resources/config/services/controller.xml [hunk 3]
```diff
@@ -3,15 +3,19 @@
 <!--
 
  This file is part of the Sylius package.
-
+sylius.controller.shop.contact
```
Comment (NoResponseMate) [inline src/Sylius/Bundle/ShopBundle/Resources/config/services/controller.xml]: 💃

**Review by GSadee — APPROVED** [review]

**Comment by GSadee** [comment]
Thanks, Karol! :tada:

**Pull request merged**